src/interaction/ArcBallCamera.cpp

```diff
@@ -0,0 +1,250 @@
+//  ----------------------------------
+//  Copyright � 2017, Brown University, Providence, RI.
+//  
+//  All Rights Reserved
+//   
+//  Use of the software is provided under the terms of the GNU General Public License version 3 
+//  as published by the Free Software Foundation at http://www.gnu.org/licenses/gpl-3.0.html, provided 
+//  that this copyright notice appear in all copies and that the name of Brown University not be used in 
+//  advertising or publicity pertaining to the use or distribution of the software without specific written 
+//  prior permission from Brown University.
+//  
+//  See license.txt for further information.
+//  
+//  BROWN UNIVERSITY DISCLAIMS ALL WARRANTIES WITH REGARD TO THIS SOFTWARE WHICH IS 
+//  PROVIDED �AS IS�, INCLUDING ALL IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS 
+//  FOR ANY PARTICULAR PURPOSE.  IN NO EVENT SHALL BROWN UNIVERSITY BE LIABLE FOR ANY 
+//  SPECIAL, INDIRECT OR CONSEQUENTIAL DAMAGES OR FOR ANY DAMAGES WHATSOEVER RESULTING 
+//  FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN ACTION OF CONTRACT, NEGLIGENCE OR 
+//  OTHER TORTIOUS ACTION, OR ANY OTHER LEGAL THEORY, ARISING OUT OF OR IN CONNECTION 
+//  WITH THE USE OR PERFORMANCE OF THIS SOFTWARE. 
+//  ----------------------------------
+//  
```

Comment:
is this license still correct/needed?

Reply:
It's been there since the begging. Why not having it?

Reply:
typically we keep the license at the top level of the repo, then there's the question of author/date here as you are now authoring/editing this file as well.  I don't really have a strong preference here, was just wondering about it.

Reply:
what do you mean top of the repo? the README ?

Reply:
as a `LICENSE` file in the repo - see https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository

Reply:
I see. Then, should be MIT or GPL? The web volume viewer is MIT.

Reply:
usually we go with MIT